opentelemetry-auto-instrumentation/lib/opentelemetry-auto-instrumentation.rb

```diff
@@ -0,0 +1,153 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# OTelBundlerPatch
+module OTelBundlerPatch
+  # Nested module to handle OpenTelemetry initialization logic
+  module OTelInitializer
+    OTEL_INSTRUMENTATION_MAP = {
```

Comment:
to be clear. This will add the constant to Bundler::Runtime as well right?

Reply:
Yes, it will be part of Bundler::Runtime.
I also added another rails example for testing (with separated readme).